lib/fog/voxel.rb

```diff
@@ -1,7 +1,7 @@
-require 'fog/voxel/version'
 require 'fog/core'
 require 'fog/xml'
 require 'digest/md5'
+require File.expand_path('../voxel/version', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.